FLEX.xcodeproj/project.pbxproj

```diff
@@ -2126,9 +2130,9 @@
 					"$(PROJECT_DIR)",
 				);
 				INFOPLIST_FILE = Classes/Info.plist;
-				INSTALL_PATH = "@rpath";
+				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
 				IPHONEOS_DEPLOYMENT_TARGET = 9.0;
-				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
+				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/../Frameworks @loader_path/Frameworks";
```

Comment:
I am not sure about this change.

a typical ios app bundle should look like:
```
MyApp.app
├── MyApp
└── Frameworks
    └── FLEX.framework
        ├── FLEX
        └── Info.plist
```
`@executable_path` should refer to `MyApp.app/MyApp`, then you shouldn't need `../` here.
instead, macOS app bundle put main executable in `MyApp.app/Content/MacOS`, then `../` can make sense:
```
MyApp.app
└── Contents
    ├── Frameworks
    │   └── FLEX.framework
    │       ├── FLEX
    │       └── Info.plist
    └── MacOS
        └── MyApp
```

Reply:
You're correct. 
The root cause is that DYLIB_INSTALL_NAME_BASE should keep @rpath. 
LD_RUNPATH_SEARCH_PATHS rollbacked.